abis/mlibc/in.h

```diff
@@ -19,6 +19,7 @@ struct sockaddr_in {
 	struct in_addr sin_addr;
 	uint8_t pad[8];
 };
+#define sin_zero pad		/* for BSD Unix compatibility */
```

Comment:
How do applications use this? (For example, do they care about the size of this field?)

Reply:
they don't, probably, but iirc you should zero it, which is what `glib` is doing, however, they call it via sin_zero, linux solves that issue by defining it